fastlane_bot/events/managers/base.py

```diff
@@ -134,12 +134,11 @@ def handle_solidly_exchanges(self, exchange):
         Handles getting stable & volatile fees for Solidly forks
         """
         exchange_name = exchange.exchange_name
-        self.factory_contracts[exchange_name] = self.web3.eth.contract(
+        self.factory_contracts[exchange_name] = self.w3_async.eth.contract(
```

Comment:
This change might lead to the following problem:

In the bancor_v3 exchange, [function get_fee is not declared 'async'](https://github.com/bancorprotocol/fastlane-bot/blob/main/fastlane_bot/events/exchanges/bancor_v3.py#L41), however, it is [called with 'await' for ALL exchanges](https://github.com/bancorprotocol/fastlane-bot/blob/main/fastlane_bot/events/async_event_update_utils.py#L72):
```
fee = await ex.get_fee(address, contract)
```
Which means that this function probably has to be declared `async`.